PartyIcons/View/NameplateView.cs

```diff
@@ -224,25 +286,51 @@ public void SetupForPC(XivApi.SafeNamePlateObject npObject)
             }
         }
 
-        private int GetClassIcon(XivApi.SafeNamePlateInfo info)
+        private int GetClassIcon(XivApi.SafeNamePlateInfo info, int def = -1)
         {
+            if (def != -1 && !ignorables.Contains(def))
+                return def;
+        	
             var genericRole = JobExtensions.GetRole((Job)info.GetJobID());
             var iconSet = _stylesheet.GetGenericRoleIconset(genericRole);
             return _iconSet.GetJobIcon(iconSet, info.GetJobID());
         }
 
-        private IntPtr GetStateNametext(int iconId)
+        private bool IsIgnorableStatus(int statusIcon)
```

Comment:
'Ignorable' is a bad choice of terminology here. It may be ignorable, but who decides whether it is actually ignored? If it's this very method then it should be stated that way. Also it's not the status that is being ignored, but the icon, so that word should be included also.